notebooks/RAG_zephyr_langchain.ipynb

```diff
@@ -0,0 +1,11589 @@
+{
```

Comment:
<p>On your last line: you could mention the method <code>tokenizer.apply_chat_template</code> to convert a list of messages (as dicts: <code>{'role': 'user', 'content': '(...)'}</code>) into a string with the appropriate chat format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Added a note